src/hypertrace/agent/config/config_pb2.py

```diff
@@ -1,7 +1,8 @@
-# -*- coding: utf-8 -*-
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
 # source: config.proto
-"""Generated protocol buffer code."""
+
```

Comment:
I am curious about the reason for this change. If it is not due to a new field I would avoid it until we sort out the proto compilation issue https://github.com/hypertrace/pythonagent/issues/165.

Reply:
the file was regenerated as part of the standard build process. At some point, one of us added a comment to the top of the generated file to get pylint and pydocs to be happy. Intestingly, it isn't complaining about it missing anymore.

Reply:
Adding the comment back in.

Reply:
done.

Reply:
I am sorry I wasn't clear. I was more talking about the file itself rather than specific line. Not a big deal.